snapshots/overlay/overlay_test.go

```diff
@@ -306,11 +482,12 @@ func testOverlayView(t *testing.T, newSnapshotter testsuite.SnapshotterFunc) {
 	if m.Source != expected {
 		t.Errorf("expected source %q but received %q", expected, m.Source)
 	}
+
 	if m.Options[0] != "ro" {
-		t.Errorf("expected mount option ro but received %q", m.Options[0])
+		t.Errorf("expected mount option ro but received %q", m.Options[1])
```

Comment:
why it is `1`?

Reply:
Ouch, writing tests at 3 am sometimes is not a good idea :D